notebook/static/notebook/less/outputarea.less

```diff
@@ -158,13 +158,14 @@ div.output_javascript:empty {
 
 div.raw_input_container {
     font-family: @font-family-monospace;
+    line-height: @code_line_height;
     // for some reason, em padding doesn't compute the same for raw_input
     // that is not the first input, but px does
     padding-top: 5px;
 }
 
 span.raw_input_prompt {
-    /* nothing needed here */
+    white-space: pre;
```

Comment:
Should this be pre-wrap, to allow a really long prompt text to wrap? Input prompts in the terminal will wrap when they exceed the terminal width.

https://css-tricks.com/almanac/properties/w/whitespace/